### ISSUE TITLE
Request: Theano

### ISSUE THREAD

**Issue by sisp** [simple-icons/simple-icons#7300]
### Brand Name

Theano

### Website

https://github.com/Theano/Theano

### Popularity Metric

GitHub Stars: 9.6k

### Official Resources for Icon and Color

svg: https://github.com/Theano/Theano/blob/8fd9203edfeecebced9344b0c70193be292a9ade/doc/images/theano_logo_allblue.svg
color: `#11557c`

I suggest to extract only the `e` in Th_e_ano as the icon.

### Additional Comments

I'd like to check first whether Theano would be accepted.

Theano is a piece of modern ML/AI software history. To the best of my knowledge, Theano was the first open-source software in the AI/ML space using a flavor of symbolic programming and computation graphs which enabled a tremendous degree of flexibility in designing neural-network-like models thanks to symbolic differentiation. A large body of ML research over a decade or so used Theano. Its unique approach at the time laid the foundation for numerous high-level APIs like Keras and eventually TensorFlow, which took off as essentially a "Theano 2.0" with Keras now being its official primary API.

BUT: Theano is no longer being developed. There exists a fork under the name [Aesara](https://github.com/aesara-devs/aesara), but compared to TensorFlow, PyTorch, etc. with huge communities and corporate backing, I believe Aesara mainly has a nostalgic touch.

That said, would you accept a PR with the Theano icon?

### Contributing

Yes

**Comment by sachinraja** [comment]
I would say no since we typically remove brands when they no longer exist or are no longer active. I really don't like that we have to do this though. I wish there was a way for us to version each icon independently.

**Comment by sisp** [comment]
I'm not sure I understand your point. Would you like to keep icons of inactive brands? Or are you unhappy with versioning the whole icon collection as one because removing an icon means a breaking change which translates to a major version bump with SemVer?

**Comment by sachinraja** [comment]
> Would you like to keep icons of inactive brands?

Yes, because our users may still be using those.

> Or are you unhappy with versioning the whole icon collection as one because removing an icon means a breaking change which translates to a major version bump with SemVer

I'm unhappy with it but not for that reason. I think it makes sense to remove "dead" brands when we have a single big package for all icons. IMO it make sense for each *icon* to be released as a separate npm package but this is not ideal at all for users who would have to install a separate package for each icon.

**Comment by sisp** [comment]
This is getting a bit off-topic, so we might want to create a dedicated issue.

> IMO it make sense for each icon to be released as a separate npm package but this is not ideal at all for users who would have to install a separate package for each icon.

It might be possible to have the best of both worlds: Each icon gets its own package and there could be a meta package which depends on all icon packages of active brands. The dependencies of the meta package could be updated automatically using Dependabot or Renovate Bot. When a brand becomes inactive, its individual icon package can remain but it gets removed from the meta package's dependencies.

What do you think?

**Comment by sachinraja** [comment]
I like that solution, but, unfortunately, I don't think we can have a meta package. It would have over 2,000 dependencies.

**Comment by sisp** [comment]
I'm closing this issue since it's off-topic here and you've created a dedicated issue for the package discussion.
